package.json

```diff
@@ -37,8 +37,6 @@
   },
   "babel": {
     "plugins": [
-      "lodash",
-      "transform-object-assign",
```

Comment:
We don't use Object.assign anywhere in our client-side code so we can remove this babel plugin too.